src/test/scala/com/vngrs/etl/CombineSpec.scala

```diff
@@ -0,0 +1,62 @@
+package com.vngrs.etl
+
+/**
+  * Combiner Specifications
+  */
+class CombineSpec extends BaseSparkSpec {
```

Comment:
I still think this one is a bad abstraction. Can you show me any case it is useful?